README.md

```diff
@@ -87,7 +87,10 @@ The license is Apache 2.0, see LICENSE-2.0.txt.
 
 ### Binary Releases
 
-You can find published releases (compiled for Java 6 and above) on
+Version 1.2.1 and earlier were built for Java 6, while newer
+versions will be built for Java 8.
```

Comment:
OK.  I was going to say then maybe there could be one or two more additional releases off of a 1.2.x branch that didn't include this; then I realized that the change you've been helping with is probably also way too big to go into a 1.2.2 :)